README.rst

```diff
@@ -25,7 +25,7 @@ for versioning.
 Support Guarantees
 ------------------
 
-* Major branch development period: undefined
+* Major branch development period: 6 months
```

Comment:
And this is based on what? :)
I think 6 months could be a bit stretched for Makd, is just a build thing, it should pose very little risk to upgrade, so I would go for something more in the lines of 3 months.

Reply:
I just went with what ocean (and IIRC other core team projects) is doing. I'm fine with 3 months and will update to that.